docs/csharp/programming-guide/concepts/linq/linq-and-generic-types.md

```diff
@@ -32,5 +32,5 @@ ms.assetid: 660e3799-25ca-462c-8c4a-8bce04fbb031
   
 ## See also
 
-- [Getting Started with LINQ in C#](../../../../csharp/programming-guide/concepts/linq/getting-started-with-linq.md)
+- [Getting Started with LINQ in C#](../../../../csharp/programming-guide/concepts/linq/index.md)
```

Comment:
The same comment here as earlier. This link was present to meet the requirement of linking to the parent topic. It can either be removed or the link description changes.